packages/api/test/utils/checkAgainstSpec.ts

```diff
@@ -5,31 +5,66 @@ import {applyRecursively, OpenApiJson, parseOpenApiSpec, ParseOpenApiSpecOpts} f
 import {GenericServerTestCases} from "./genericServerTest.js";
 
 const ajv = new Ajv({
-  // strict: true,
-  // strictSchema: true,
   allErrors: true,
 });
 
-// TODO: Still necessary?
+// Ensure embedded schema 'example' do not fail validation
```

Comment:
Schemas from the beacon chain embed `example` sections. Those are not valid json schema AFAIK and fail validation.
This essentially ignores those properties.

Reply:
My comment was specifically regarding if we still need to add `example` as keyword, this is related to previous TODO comment (`TODO: Still necessary?`) because I remember that at some point ajv removed `example` from default keywords. See related issue https://github.com/loopbackio/loopback-next/issues/9308, the workaround is exactly what we did here to add the keyword manually.

> This essentially ignores those properties.

I'd assume the examples should be valid, if not seems an issue with the spec?


Or is this related to removing this 👇 
```
validate: () => true,
errors: false,
```

Was is not validated before but we validate examples now?

Reply:
> I'd assume the examples should be valid, if not seems an issue with the spec?

You can ignore this question, I remember now that as per openapi spec examples are allowed but ajv does not have them, that's why we need custom keyword.

I would assume that by default it does not validate if this still works after removing
```
validate: () => true,
errors: false,
```

Reply:
It doesn't validate the content of `example` but allows this property to exist (w/o this call validation fails with e.g. `request schema - strict mode: unknown keyword: "example"`).

> per openapi spec examples 

Ah right, i's valid `openapi` but not `JSON Schema`. As in our tests the whole openapi `schema` is used for `JSON Schema` validation it fails because of this extra property.
Probably the better solution is to remove those examples from `schema` before validation.
Current solution allows `example` to be anywhere, might be problematic.

Reply:
> Probably the better solution is to remove those examples from schema before validation.
Current solution allows example to be anywhere, might be problematic.

Can't we just keep the previous behavior so that it is considered always valid (`validate: () => true`)?

Reply:
👍